lib/zmq-kernel.coffee

```diff
@@ -0,0 +1,343 @@
+child_process = require 'child_process'
+path = require 'path'
+
+_ = require 'lodash'
+jmp = require 'jmp'
+uuid = require 'uuid'
+zmq = jmp.zmq
+
+Kernel = require './kernel'
+
+
+module.exports =
+class ZMQKernel extends Kernel
+    constructor: (kernelSpec, @grammar, @config, @configPath, onlyConnect = false) ->
```

Comment:
`@onlyConnect` needs to be an instance property, otherwise `@destroy` won't work.